db/seeds.rb

```diff
@@ -47,9 +47,9 @@
 ronald4 = User.find(4)
 
 # tenants
-Tenant.create!(
+bridge_builders = Tenant.create!(
   location:              "East Timor represent",
-  organization:          "Bridge Builders off the island."
+  organization:          "bridge-builders"
```

Comment:
since this branch doesn't relate to doing that, I think that's something we can do in another branch.